### PR TITLE
Remove workflow of collection blocks.

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

**Comment by jone** [comment]
@mbaechtold I would rephrase that to "remove workflow for collection blocks", because it actually does no longer have a workflow.
The permissions are acquired, not the workflow.
It did confuse me for a second 😂 
It's a little picky, sorry

**Comment by mbaechtold** [comment]
Much better. Thank you, @jone!

**Comment by jone** [comment]
@mbaechtold I've just realized that this package is already published on pypi; I think we need to have an upgrade step 😕 
Be aware that the security of the collection blocks need to be re-set so that all permissions are acquired.
Here is an example: https://github.com/4teamwork/winterthur.web/pull/159/

**Comment by mbaechtold** [comment]
@jone, I have added an upgrade step!

**Comment by jone** [comment]
Awesome 👍
